lib/interviewer/containers/Interfaces/FinishSession.js

```diff
@@ -1,23 +1,70 @@
-import { useEffect, useState } from 'react';
+import { useState, useEffect, useCallback } from 'react';
 import { useDispatch } from 'react-redux';
 import Button from '~/lib/ui/components/Button';
-import FinishInterviewModal from '~/app/(interview)/interview/_components/FinishInterviewModal';
+import { actionCreators as dialogActions } from '../../ducks/modules/dialogs';
```

Comment:
Would be nice to fix the import path here.

Reply:
resolved [7e84cd5](https://github.com/complexdatacollective/Fresco/pull/82/commits/7e84cd559e6db0a8cf986b0cbd2f758c79a3cca7)